__tests__/app.test.js

```diff
@@ -37,6 +37,43 @@ describe('GET /api/topics', () => {
         })
     })
 })
+describe('GET /api/articles', () => {
+    test('get /api/articles request responds with articles object containing an array of articles objects', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(response => {
+            const resBodyArticles = response.body.articles;
+            expect(resBodyArticles.length).toBe(12)
+            resBodyArticles.forEach(article => {
+                console.log(typeof article.created_at)
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(Number)
+
+                })
+            })
+        })
+    })
+    test('get /api/articles request returns array sorted by date created in ascending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(response => {
```

Comment:
Valid test for sort order, HOWEVER It would have been far easier to do this with jest-sorted